DEVELOPING.md

```diff
@@ -52,22 +24,16 @@ make style
 make quality
 ```
 
-This will run automatic code styling using `black` and `isort` and test that the
+This will run automatic code styling using `ruff`, `flake8`, `black`, `isort`, and `mypy` to test that the
 repository's code matches its standards.
 
 **EXAMPLE: test changes locally**
 
 ```bash
-make test TARGETS=<CSV of frameworks to run>
+make test
 ```
 
-This will run the targeted SparseML unit tests for the frameworks specified.
-The targets should be specified, because not all framework dependencies can be installed to run all tests.
-
-To run just PyTorch tests, run
-```bash
-make test TARGETS=pytorch
-```
+This will run the targeted LLM Compressor unit tests for the frameworks specified.
```

Comment:
We should also mention how to run the nightly/weekly tests by setting the CADENCE environment variable: `export CADENCE="nightly"`

Reply:
I'd like to expand on this further once we have the test migration in a more finalized state. For now, wanted to keep it generic without over-committing to any pathways as that gets resolved

Reply:
Sure, thats good with me